products/apache-nifi.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Apache NiFi
+category: server-app
+tags: apache java-runtime
+iconSlug: apachenifi
+permalink: /apache-nifi
+alternate_urls:
+-   /nifi
+changelogTemplate: "https://cwiki.apache.org/confluence/display/NIFI/Release+Notes#ReleaseNotes-Version__LATEST__"
+eolColumn: Support
+releaseDateColumn: true
+
+identifiers:
+-   repology: nifi
+
+auto:
+  methods:
+  -   git: https://github.com/apache/nifi.git
```

Comment:
```suggestion
  -   git: https://github.com/apache/nifi.git
      regex: '^rel\/nifi-(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)$'
```

The pattern used on https://github.com/apache/nifi/tags does not match the default pattern.